packages/actions/log/src/components/StyledComponents.js

```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components'
+import {theme, declareFont} from 'tocco-ui'
+
+export const StyledContainer = styled.div`
```

Comment:
todo: I think we need to check with @yilkur so that we do not have all the hardcoded values here.

Reply:
@LorenzKieran for padding and margin you should use the scale functions. For example `padding-right: ${scale.space(-0.5)};`. 

For colors use only theme colors. For example white `background-color: ${themeSelector.color('paper')};`

You can find a lot of examples within existing styling :)